k8s/upgrades/0.8.2-0.9.0/cstor/cstor-pool-update-082-090.yaml

```diff
@@ -0,0 +1,664 @@
+# CASTemplate cstor-pool-update-082-090 is
+# used to upgrade single cstor pool
+apiVersion: openebs.io/v1alpha1
+kind: CASTemplate
+metadata:
+  name: cstor-pool-update-082-090
+spec:
+  defaultConfig:
+  - name: baseversion
+    value: "0.8.2"
+  - name: targetversion
+    value: "v0.9.0-RC1"
+  - name: successStatus
+    value: "Success"
+  - name: failStatus
+    value: "Fail"
+  run:
+    tasks:
+    # Gets cstorpool cr for given cstorpool
+    - upgrade-cstor-pool-082-090-get-cstorpool
+    # Gets cstor pool deployment for given cstorpool
+    - upgrade-cstor-pool-082-090-get-deployment
+    # Gets storgepool for given cstorpool
+    - upgrade-cstor-pool-082-090-get-storagepool
+    # puts result og resource getting via above 3 runtasks
+    - upgrade-cstor-pool-082-090-patch-upgrade-result
+    # patches pool deployment with target version
+    - upgrade-cstor-pool-082-090-patch-deployment-image
+    # Waits until new pods have came up successfully
+    - upgrade-cstor-pool-082-090-patch-deployment-image-status
+    # checks whether pool containers in target version or not
+    - upgrade-cstor-pool-082-090-post-check-patch-deployment-image
+    # lists all replicasets of pool deployment
+    - upgrade-cstor-pool-082-090-patch-sp-version
+    - upgrade-cstor-pool-082-090-patch-sp-version-post-check
```

Comment:
do we want to provide comments here as well?

Reply:
IMO RunTask itself describe what it does

Reply:
Lets follow one standard. Either have them or dont have them if runtask names are sufficient.